frontend/packages/console-dynamic-plugin-sdk/src/app/components/utils/resource-status.tsx

```diff
@@ -1,6 +1,6 @@
 import * as React from 'react';
-import * as classNames from 'classnames';
 import { Badge } from '@patternfly/react-core';
+import * as classNames from 'classnames';
 
 export const ResourceStatus: React.FC<ResourceStatusProps> = ({
```

Comment:
we can have `ResourceStatus` as the default export over named one's and class names added in this component are like `co-resource-item__resource-status`  where `co` is for the console.   As we migrate to SDK shall we handle these as well?

cc @andrewballantyne

Reply:
Agreed on the default export front.

The renaming of classNames is weird the way Console is coded... some of these classes belong to the global css file and we could dupe them, but I wouldn't "move" (rename) them. Once we are away from OpenShift Console and in the SDK repo, we'll need to get access to these classes so I think we have to dupe them and co-locate them -- which we might as well do as part of this PR. 

Good catch.